pkg/minikube/registry/drvs/hyperkit/hyperkit.go

```diff
@@ -85,5 +88,57 @@ func status() registry.State {
 		return registry.State{Installed: true, Error: fmt.Errorf("%s failed:\n%s", strings.Join(cmd.Args, " "), out), Fix: "Run 'brew install hyperkit'", Doc: docURL}
 	}
 
+	// Split version from v0.YYYYMMDD-HH-xxxxxxx or 0.YYYYMMDD to YYYYMMDD
+	specificVersion := splitHyperKitVersion(version.GetHyperKitVersion())
+	// If hyperkit is not newer than specificVersion, suggest upgrade information
+	isNew := isNewerVersion(convertVersionToDate(string(out)), specificVersion)
+	if !isNew {
+		return registry.State{Installed: true, Error: fmt.Errorf("%s is too old. the higher than %s is needed", "hyperkit version", version.GetHyperKitVersion()), Fix: "Run 'brew upgrade hyperkit' or Upgrade Docker for Desktop version", Doc: docURL}
+	}
+
 	return registry.State{Installed: true, Healthy: true}
 }
+
+// isNewerVersion checks whether current hyperkit is newer than specific version
+func isNewerVersion(currentVersion string, specificVersion string) bool {
+	// Convert hyperkit version to time.Date to compare whether hyperkit version is not old.
+	layout := "20060102"
+	currentVersionDate, err := time.Parse(layout, currentVersion)
+	if err != nil {
+		glog.Warningf("Unable to parse to time.Date: %v", err)
+	}
```

Comment:
`isNewerVersion` should probably return the error rather than silently dropping it, so that the caller can determine what the right course of action is.

Reply:
Thanks!
I changed that `isNewerVersion` returns error if `err != nil`.